content/en/docs/features/control-plane/entity-tag.md

```diff
@@ -0,0 +1,217 @@
+participant CGW as "CoAP Gateway"
+participant RA as "Resource Aggregate"
+participant EB as "Event Bus"
+participant RD as "Resource Directory"
+
+Device -> CGW ++: Sign In
+CGW ->Device: Signed In
+CGW -> RA++: UpdateDeviceMetadataRequest with\nTwinSynchronization.State = SYNCING
+RA -> EB: DeviceMetadataUpdated with\nTwinSynchronization.State = SYNCING
+RA -> CGW
+deactivate RA
+CGW -> RD++: Get latest ETAG from the all device resources
+RD -> Database: Get latest ETAG from the all device resources
+Database -> RD: The latest ETAG
+RD -> CGW: The latest ETAG
+deactivate RD
+CGW -> Device++: Batch observe resources with the latest ETAG
+alt Latest ETAG == highest ETAG among all device resources
+Device -> CGW: Respond with code VALID
+else Latest ETAG != highest ETAG among all device resources
+Device -> CGW: Current representation of resources with the current ETAG
+CGW -> RA++: BatchNotifyResourceChangedRequest with the current ETAG
+  RA --> Database: Store ResourceChanged event that contains the current ETAG for each resource
+  RA --> EB: Publish ResourceChanged event that contains the current ETAG for each resource
+return BatchNotifyResourceChangedResponse
+end
+CGW -> RA++: UpdateDeviceMetadataRequest with\nTwinSynchronization.State = IN_SYNC
+RA -> EB: DeviceMetadataUpdated with\nTwinSynchronization.State = IN_SYNC
+RA -> CGW: UpdateDeviceMetadataResponse
+deactivate RA
+
+== Resource content changed ==
+
+Device -> CGW : [NOTIFY] 'oic.r.temperature' changed with the current ETAG
+CGW -> RA++: NotifyResourceChangedRequest with the current ETAG
+RA -> Database: Store ResourceChanged event that contains the current ETAG
+RA -> EB: Publish ResourceChanged event that contains the current ETAG
+
+@enduml
+{{< /plantuml >}}
+
+In order to monitor resource changes and determine if a resource has been modified on the device, the CoAP gateway utilizes the Entity Tag (ETAG) mechanism.
+
+For **Batch Observation**, the ETAG is associated with the overall state of resources. Prior to initiating resource observation, the CoAP gateway retrieves the latest ETAG among all device resources from the Hub Database. During resource observation initiation, the CoAP gateway sends the ETAG to the device. If the received ETAG matches the highest ETAG among the device resources, the device responds with a code `VALID`. However, if the received ETAG does not match, the device responds with a code `CONTENT` and includes the current ETAG. Consequently, when a resource changes, the device sends the updated ETAG back to the CoAP gateway via a notification. The CoAP gateway transmits the ETAG together with the Content by using the `NotifyResourceChanged` method to the resource-aggregate. This command is then converted into a `ResourceChanged` event, which is saved in a database and distributed through the event bus. In cases where multiple resources change simultaneously, the CoAP gateway updates all affected resources with the same timestamp and ETAG.
```

Comment:
During the resource observation initiation -> when initiating the res obs

Reply:
fixed